src/components/Dashboard/TodoList.vue

```diff
@@ -108,7 +124,7 @@
               </div>
             </div>
 
-            <div class="list-item__actions">
+            <div class="list-item__actions" v-if="!isCorrection(item)">
```

Comment:
We realized today that we will need the ability to Resume a draft correction filing (although I haven't implemented that yet).

Reply:
Oh, interesting!

Reply:
Shouldn't be too bad... You'll need to add a case in doResumeFiling(). You will need to pass 2 params: the id of the draft filing and the id of the corrected filing.

Right now the Correction page only looks for `this.$route.params.id` so I'll rename that param to "correctedFilingId". Use "id" for the draft's ID.